packages/test/local-server-stress-tests/src/localServerStressHarness.ts

```diff
@@ -1338,3 +1351,111 @@ export namespace createLocalServerStressSuite {
 				skip: [...seeds, ...(providedOptions?.skip ?? [])],
 			});
 }
+
+/**
+ * Operation type for cloning a client using its pending local state.
+ *
+ * This operation triggers a clone of an existing client by capturing its
+ * pending local state (via `getPendingLocalState()`) and loading a new
+ * container from it.
+ */
+interface GetClientPendingState {
+	type: "getClientPendingState";
```

Comment:
Here and elsewhere, maybe use naming that will more directly describe the action, e.g. "restartClientFromPendingState" or something